webrtc.html

```diff
@@ -625,6 +625,313 @@
           </li>
         </ol>
 
+        <p>To <dfn id=
+        "update-ice-gathering-state">update the ice gathering state</dfn> of an
+        <code><a>RTCPeerConnection</a></code> instance <var>connection</var> to
+        <var>newState</var>, the User Agent MUST queue a task that sets
+        <var>connection</var>'s <a href=
+        "#dom-peerconnection-ice-gathering-state">iceGatheringState</a>
+        attribute to <var>newState</var> and fires a simple event named
+        <code><a href=
+        "#event-icegatheringstatechange">icegatheringstatechange</a></code> at
+        <var>connection</var>.</p>
+
+        <p>To <dfn id=
+        "update-ice-connection-state">update the ice connection state</dfn> of an
+        <code><a>RTCPeerConnection</a></code> instance <var>connection</var> to
+        <var>newState</var>, the User Agent MUST queue a task that sets
+        <var>connection</var>'s <a href=
+        "#dom-peerconnection-ice-connection-state">iceConnectionState</a>
+        attribute to <var>newState</var> and fires a simple event named
+        <code><a href=
+        "#event-iceconnectionstatechange">icegatheringstatechange</a></code> at
+        <var>connection</var>.</p>
```

Comment:
Since we're replacing iceConnectionState with "connectionState" which is an aggregate state, I think we need to define these rules for changing state to be in terms of IceTranpsorts.